src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -283,12 +289,27 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
         if (this.props.onSelectedRegionsChanged) {
             this.props.onSelectedRegionsChanged([]);
         }
+        if (selectedRegions.find((r) => r.category !== selectedRegions[0].category)
+            === undefined) {
+            if (selectedRegions[0].category === RegionCategory.Checkbox) {
+                this.setTagType(inputTag[0]);
+            }
+        }
 
         this.redrawFeatures(this.imageMap.getAllFeatures());
         this.redrawFeatures(this.imageMap.getAllCheckboxFeatures());
         this.applyTagFlag = true;
     }
 
+    private setTagType = (tagInput: ITag) => {
```

Comment:
make fieldType as a parameter, because the method name is setTagType not setTagTypeToCheckbox